spec/rubocop/cop/style/block_delimiters_spec.rb

```diff
@@ -351,6 +351,14 @@
         RUBY
       end
 
+      it 'registers an offense when combined with attribute assignment' do
+        expect_offense(<<~RUBY)
+          foo.bar = baz.map { |x|
+                            ^ Avoid using `{...}` for multi-line blocks.
+          }
+        RUBY
+      end
```

Comment:
This test case is good. OTOH, `EnforcedStyle: 'semantic'`, `EnforcedStyle: 'always_braces'` and  method chained `EnforcedStyle: braces_for_chaining` should not be warned. Could you add these test cases and pass them?

Reply:
Done!